backend/src/test/admin.test.js

```diff
@@ -1,10 +1,200 @@
+const {app,postgres} = require('../main'); 
+const cleardb = `
+TRUNCATE items, bids, users, shipments, payments; 
+`;
+const addFunds = `
+    UPDATE users 
+    SET balance = (balance + $2)
+    WHERE id = $1
+    RETURNING *`;
+const adminkey = '2c6rtf7gy8bFehUVfui373undbjhvUGuhBiUEefpwid';
```

Comment:
This shouldn't be put in plain text. Use `process.env.ADMIN_TOKEN` or whatever it is

Reply:
Great point. I fixed this